File: leetcode-submissions/0344-reverse-string/solution.java
class Solution {
    public void reverseString(char[] s) {
        //aced...
        int forward = 0;
        int backward = s.length - 1;
        while(forward < backward) {
            char temp = s[backward];
            s[backward] = s[forward];
            s[forward] = temp;
            forward++;
            backward--;
        } 
    }
}

File: leetcode-submissions/0344-reverse-string/solution.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        """
        Do not return anything, modify s in-place instead.
        """
        start = 0
        end = len(s) - 1
        
        while start < end:
            temporary_string = s[start]
            
            s[start] = s[end]
            s[end] = temporary_string
            
            start += 1
            end -= 1
            

        
